src/OptionsPopup.qml

```diff
@@ -826,6 +832,40 @@ Window {
 
             addCmdline("cfg80211.ieee80211_regdom="+fieldWifiCountry.editText)
         }
+        if (chkUSBEther.checked) {
+            addConfig("dtoverlay=dwc2,dr_mode=peripheral")
+            // only required if no conf is used to load modules
+            // g_ether must not be loaded when using manual config
+            // with sh script
+            addCmdline("modules-load=dwc2,g_ether")
+            // TODO: generate mac addresses if there are issues with DHCP
+            //addCmdline("g_ether.dev_addr=8A:89:6a:8d:14:22")
+            //addCmdline("g_ether.host_addr=6A:89:6a:8d:14:22")
+            addCmdline("g_ether.idVendor=0x04b3")
+            addCmdline("g_ether.idProduct=0x4010")
+            addCmdline("g_ether.iManufacturer=\"Raspberry Pi\"")
+            addCmdline("g_ether.bcdDevice=0x0100")
+            addCmdline("g_ether.iProduct=\"USB Ethernet Gadget\"")
+            // TODO: maybe set device serial
```

Comment:
Right, I now implemented a solution using config files for the deb package. 
Maybe we can also use this approach here and then replace the serial number in the config file with the correct one.
This would require the use to reboot after first boot but it would reduce all these config values from cmdline.txt

Reply:
Implemented in last commit. And since my Pi 5 reboots multiple times after inserting a fresh setup this reboot requirement isn't noticeable.